ipaserver/install/adtrust.py

```diff
@@ -435,6 +435,9 @@ def install(standalone, options, fstore, api):
         # and propose them to be added to the list
         add_new_adtrust_agents(api, options)
 
+    # Enable configured services and update DNS SRV records
+    service.sync_services_state(api.env.host)
```

Comment:
The ``adtrust.install()`` is also called from the ``ipa-server-install`` script. This change may enable the server a bit too early if the installer is called with ``--setup-adtrust``.

Reply:
.. and also from `ipa-replica-install` too. I'll check these paths.